### PR TITLE
adding descriptions for commands and fix 'datadog host'

### PR DESCRIPTION
**Pull request by alvin-huang**

### CODE REVIEW

File: src/hubot-datadog-plus.coffee [hunk 1]
```diff
@@ -5,20 +5,26 @@
...
-#   hubot datadog host unmute <name>
-#   hubot datadog graph <1h|4h|12h|1d|1w> <memory|cpu|load|iowait|disk|disk-util> <host=foo|tag=foo:bar>
+#   hubot datadog monitor me - returns a list of monitors not in an 'OK' state
+#   hubot datadog monitor [me] <id> - returns a list of hosts triggering monitor <id> along with their state
+#   hubot datadog monitor mute <id> - mutes a monitor with <id>
+#   hubot datadog monitor unmute <id> - unmutes a monitor with <id>
+#   hubot datadog monitor mute all - mute all monitors
+#   hubot datadog monitor unmute all - unmute all monitors
+#   hubot datadog host [me] <pattern> - returns all hosts that match <pattern> 
+#   hubot datadog host mute <name> - mute/schedule downtime for a host
```
Comment (alexcreek) [inline src/hubot-datadog-plus.coffee]: so this doesnt actually schedule downtime, it just mutes.  I dont think being able to schedule downtime vs muting is actually useful but downtime is exposed in the dd api. the difference is the scoping of whats being affected and you can make it recurring.  need to talk to ray and see if there's a use for scheduling downtime from the bot. 

http://docs.datadoghq.com/api/#downtimes

File: src/hubot-datadog-plus.coffee [hunk 2]
```diff
@@ -5,20 +5,26 @@
...
+#   hubot datadog host unmute <name> - unmute/remove downtime for a host
```
Comment (alexcreek) [inline src/hubot-datadog-plus.coffee]: same here

File: src/hubot-datadog-plus.coffee (hunk 1 above)
Comment (alvin-huang) [reply src/hubot-datadog-plus.coffee]: Ah ok. I didn't know -- When I muted a host it showed up as a new entry here https://app.datadoghq.com/monitors#/downtime so I thought it was the same.

**Comment by alexcreek** [comment]
LGTM

**Pull request merged**